libs/blocks/utils/utils.js

```diff
@@ -1,3 +1,5 @@
+import { getMetadata } from '../../utils/utils.js';
```

Comment:
I had not noticed this file before.  I don't think /blocks/utils is the proper place for this.  I know we unfortunately have /libs/utils/utils.js that actually isn't meant as a general util file.

I started a PR but haven't finished where we pull these types of util functions into /libs/utils/helpers.js.  What do you think about moving this file to that?

Reply:
I created `/libs/utils/helpers.js` and moved `updateLinkWithLangRoot` there (addressed in https://github.com/adobecom/milo/pull/1101/commits/e2149877f76d990b95a22a56b442badd24f5c7b6). I didn't move the other utility functions as they might
 be referenced in consumer sites. Moving them should be announced ahead so that consumers can adapt their code if needed.